djstripe/managers.py

```diff
@@ -7,6 +7,18 @@
 from django.db import models
 
 
+class StripeObjectManager(models.Manager):
+    def exists_by_json(self, data):
+        """
+        Search for a matching stripe object based on a Stripe object
+        received from Stripe in JSON format
+        :param data: Stripe event object parsed from a JSON string into an object
+        :type data: dict
+        :rtype: bool
+        """
```

Comment:
Add a line break before the parameter list and add ``:returns: True if the dj-stripe object exists, False otherwise.``

Also, I believe the first "stripe" in the docstring should be "dj-stripe"